app/src/main/res/layout/main_fragment.xml

```diff
@@ -74,6 +74,15 @@
                 app:layout_constraintStart_toStartOf="@id/start_guideline"
                 app:layout_constraintTop_toBottomOf="@id/settings_button" /-->
 
+            <androidx.appcompat.widget.SwitchCompat
+                android:id="@+id/default_queues_switch"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="Default queues "
+                app:layout_constraintTop_toBottomOf="@id/settings_button"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toEndOf="parent" />
+
```

Comment:
Could you please add a screenshot to PR description?
In my opinion, it's better to move this switch somewhere to the settings screen to avoid Main screen overloading. In fact, this switch is a setting, so it's logical to put it there, I think.

Reply:
I decided sometimes we want to start one engagement with a specific queue and immediately another without.

Reply:
Of course, I can move it to the setting screen.

Reply:
OK, I decided to move this toggle to the settings.